src/hooks/use-snapshots.tsx

```diff
@@ -280,13 +284,16 @@ export const SnapshotProvider: React.FC< { children: ReactNode } > = ( { childre
 		}
 		setIsLoading( true );
 		try {
-			const response: { site_count: number; site_limit: number } = await client.req.get( {
-				path: '/jurassic-ninja/usage',
-				apiNamespace: 'wpcom/v2',
-			} );
+			const response: { site_count: number; site_limit: number; site_creation_blocked: boolean } =
```

Comment:
We could also add the property `site_creation_blocked` to the type definition of the function `fetchSnapshotUsage`.

https://github.com/Automattic/studio/blob/5b834ea553f5097dc9caeee0fb6baecb4994ba52/src/hooks/use-snapshots.tsx#L36

Reply:
Added in https://github.com/Automattic/studio/pull/354/commits/51eda6825b2d8ca60bb6aa07849dc8ae7dce1748